src/courtroom.cpp

```diff
@@ -3233,27 +3233,33 @@ void Courtroom::on_ooc_return_pressed()
     ui_ooc_chat_message->clear();
     return;
   }
-  else if (ooc_message.startsWith("/save_chatlog"))
+  else if (ooc_message.startsWith("/save_ic") || ooc_message.startsWith("/save_ooc"))
   {
-    QFile file("chatlog.txt");
-
-    if (!file.open(QIODevice::WriteOnly | QIODevice::Text | QIODevice::Truncate))
+    bool is_ooc;
+    if (ooc_message.startsWith("/save_ic"))
+      is_ooc = false;
+    else
+      is_ooc = true;
```

Comment:
```suggestion
    bool is_ooc = !ooc_message.startsWith("/save_ic");
```

Reply:
Regarding negations: that was mainly to avoid confusion, since code that handles IC saving comes before the OOC variant elsewhere.